modules/cnv_upload_virtctl.adoc

```diff
@@ -10,24 +10,35 @@ associated with a virtual machine after the upload has completed.
 * A virtual machine disk image, in RAW or QCOW2 format. It can be compressed
 using *xz* or *gzip*.
 * *kubevirt-virtctl* must be installed on the client machine.
+* The client machine must be configured to trust the OpenShift cluster's
+certificate.
 
 .Procedure
 
 . Identify the following items:
 * File location of the VM disk image you want to upload
 * Name and size desired for the resulting PVC
 
+. Create a secured route using re-encryption termination:
```

Comment:
Before this step:

. Remove the existing passthrough route:
oc delete route -n cdi cdi-uploadproxy-route

Reply:
@aglitke why at all to create that route with kubevirt-ansible if one needs to delete it and replace with a secure one? why not add a creation of a secure one during installation?

Reply:
Because the release is eminent and we are not making any more code changes and rebuilds for this release.

Reply:
yes, obviously not in this release, but what about creating encrypted route in the next release?